### PR TITLE
DMP-3782 and DMP-3707 - add missing DB keys

### PR DESCRIPTION
**Pull request by cmagowan-hmcts**
https://tools.hmcts.net/jira/browse/DMP-3782
and 
https://tools.hmcts.net/jira/browse/DMP-3707 (Convert all non-integer courtrooms to UPPER case)

### CODE REVIEW

File: src/integrationTest/resources/tests/transcriptions/transcription/expectedResponse.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "case_number": "1",
-  "courthouse_id" : $COURTHOUSE_ID,
+  "courthouse_id": $COURTHOUSE_ID,
   "courthouse": "some-courthouse",
```
Comment (karen-hedges) [inline src/integrationTest/resources/tests/transcriptions/transcription/expectedResponse.json:4]: Should this be uppercase?

File: src/integrationTest/resources/tests/transcriptions/transcription/expectedResponseNoCourtroom.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
   "case_number": "1",
-  "courthouse_id" : $COURTHOUSE_ID,
+  "courthouse_id": $COURTHOUSE_ID,
   "courthouse": "some-courthouse",
```
Comment (karen-hedges) [inline src/integrationTest/resources/tests/transcriptions/transcription/expectedResponseNoCourtroom.json:4]: Should this be uppercase?

File: src/integrationTest/resources/tests/transcriptions/transcription/expectedResponseNoHearing.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
   "case_number": "1",
-  "courthouse_id" : $COURTHOUSE_ID,
+  "courthouse_id": $COURTHOUSE_ID,
   "courthouse": "some-courthouse",
```
Comment (karen-hedges) [inline src/integrationTest/resources/tests/transcriptions/transcription/expectedResponseNoHearing.json:4]: Should this be uppercase?

File: src/integrationTest/resources/tests/transcriptions/transcription/expectedResponseNoUrgency.json [hunk 4]
```diff
@@ -1,23 +1,25 @@
...
-  "courthouse" : "some-courthouse",
-  "case_reporting_restrictions" : [ ],
-  "status" : "Approved",
-  "received" : "2023-06-20T10:00:00Z",
-  "requestor" : {
-    "user_id" : $USER_ACCOUNT_ID,
-    "user_full_name" : "integrationtest.user@example.comFullName"
+  "case_number": "1",
+  "courthouse_id": $COURTHOUSE_ID,
+  "courthouse": "some-courthouse",
```
Comment (karen-hedges) [inline src/integrationTest/resources/tests/transcriptions/transcription/expectedResponseNoUrgency.json:4]: Should this be uppercase?

File: src/integrationTest/resources/tests/transcriptions/transcription/expectedResponse.json (hunk 1 above)
Comment (cmagowan-hmcts) [reply src/integrationTest/resources/tests/transcriptions/transcription/expectedResponse.json:4]: no, this is showing the display_name, so wont be upper-case, as per https://tools.hmcts.net/jira/browse/DMP-3651

File: src/integrationTest/resources/tests/transcriptions/transcription/expectedResponseNoHearing.json (hunk 3 above)
Comment (cmagowan-hmcts) [reply src/integrationTest/resources/tests/transcriptions/transcription/expectedResponseNoHearing.json:4]: no, this is showing the display_name, so wont be upper-case, as per https://tools.hmcts.net/jira/browse/DMP-3651

File: src/integrationTest/resources/tests/transcriptions/transcription/expectedResponseNoUrgency.json (hunk 4 above)
Comment (cmagowan-hmcts) [reply src/integrationTest/resources/tests/transcriptions/transcription/expectedResponseNoUrgency.json:4]: no, this is showing the display_name, so wont be upper-case, as per https://tools.hmcts.net/jira/browse/DMP-3651

**Review by karen-hedges — APPROVED** [review]

**Pull request merged**